src/components/RecommendedFavorites/RecommendedFavorites.component.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import {
+  StyledMainContainer,
+  StyledListGrid,
+  StyledVideoGrid,
+  StyledVideoImage,
+  StyledVideoName,
+  StyledVideoDescription,
+  StyledVideoInformation,
+  StyledVideoRow,
+} from './styled';
+
+function RecommendedFavorites({ relatedVideos }) {
+  const videoRole = 'recommended-video';
```

Comment:
Consider renaming these variables to `videoTestId` or something more appropriate for its purpose. Since it is no longer a role, the name is a bit misleading.